### PR TITLE
docs(subscriptions): add doc for 'support:app:' Stripe product metadata

### PR DESCRIPTION
**Pull request by chenba**
Doc for subscription support form field added in FXA-2434 (https://github.com/mozilla/fxa/issues/6220).

### CODE REVIEW

File: docs/fxa-engineering/fxa-sub-platform.md [hunk 1]
```diff
@@ -78,6 +78,7 @@ If you are using a new Stripe account, you will need to setup a product and its
 | product:privacyNoticeURL:{locale} | Localized override URL for the webpage containing the Privacy Notice for the product offering|
 | product:privacyNoticeDownloadURL | The URL for a downloadable version of the Privacy Notice for the product offering|
 | product:privacyNoticeDownloadURL:{locale} | Localized override URL a downloadable version of the Privacy Notice for the product offering|
+| support:app:{x} | Optional. An app or service for the support form. The form options will be in the same order as the metadata. These values shouldn't be too long as they are displayed in dropdown options of limited width.|
```
Comment (johngruen) [inline docs/fxa-engineering/fxa-sub-platform.md:81]: should {x} be {n} here

Comment (clouserw) [reply docs/fxa-engineering/fxa-sub-platform.md]: ping @chenba

Comment (chenba) [reply docs/fxa-engineering/fxa-sub-platform.md]: The newer commit explains what values `{x}` can be, which is not strictly numbers.

**Review by bbangert — APPROVED** [review]

**Comment by bbangert** [comment]
@chenba looks like there's a conflict now.

**Pull request merged**